src/elf/resolver.rs

```diff
@@ -80,7 +80,7 @@ impl ElfResolver {
         let loaded_address = if e_type == ET_EXEC {
             low_addr as Addr
         } else {
-            loaded_address
+            loaded_address + low_off as usize
```

Comment:
the whole assumption that there is a single foff_to_virt is wrong :( we talked about this offline, I'm afraid this whole logic has to be reworked

Reply:
but specifically this `loaded_address + low_off` also seems wrong. For ET_DYN load address should be zero, and low_off should be zero as well, I think...

Reply:
Yeah, I wrote that before our discussion and forgot to rework later. Will adjust.